apps/examples/app/examples/form/error-summary/demo.tsx

```diff
@@ -0,0 +1,25 @@
+import { Input, ErrorSummary, ErrorSummaryError } from "@postenbring/hedwig-react";
+import { useRef } from "react";
+import type { ExampleConfig } from "../..";
+
+function Example() {
+  const ref1 = useRef<HTMLInputElement>(null);
+  const ref2 = useRef<HTMLInputElement>(null);
+  const errors: ErrorSummaryError[] = [
+    { message: "Feil i input", ref: ref1 },
+    { message: "Feil i input", ref: ref2 },
+  ];
+  return (
+    <div>
+      <ErrorSummary heading={"Test feiloppsummering"} errors={errors} />
+      <Input label="Test 1" name="test1" ref={ref1} errorMessage={errors[0].message} />
+      <Input label="Test 2" name="test2" ref={ref2} errorMessage={errors[1].message} />
+    </div>
+  );
```

Comment:
It would be awesome to have a full demo with a submit to trigger this, but is that overkill?

Reply:
Not at all. The biggest hurdle is creating the demo. Just create a new file, e.g. `complete-form.tsx` or something and implement 👍

Reply:
I tried, but couldn't get a functional form in the demos. Not sure why that is, the demo framework started showing lots of errors when I edited the Inputs :(